FFMpegCore/FFMPEG/FFMpeg.cs

```diff
@@ -436,14 +436,21 @@ public VideoInfo ReplaceAudio(VideoInfo source, FileInfo audio, FileInfo output,
 
         public VideoInfo Convert(ArgumentContainer arguments)
         {
-            var args = ArgumentBuilder.BuildArguments(arguments);
             var output = ((OutputArgument)arguments[typeof(OutputArgument)]).GetAsFileInfo();
+            var sources = ((InputArgument)arguments[typeof(InputArgument)]).GetAsVideoInfo();
+
+            // Sum duration of all sources
+            _totalTime = TimeSpan.Zero;
+            foreach (var source in sources)
```

Comment:
Perhaps use  `_totalTime = sources.Sum(s => s.Duration)`?

Reply:
TimeSpans can not be implicitly converted to long, But we could do
`_totalTime = new TimeSpan(sources.Sum(s => s.Duration.Ticks));`

Reply:
Ah okay, I see. Just keep it as it is then :)